recipes-omnect/initrdscripts/omnect-os-initramfs/flash-mode-2

```diff
@@ -44,7 +46,7 @@ run_bmap_tool() {
         fi
     done </tmp/all_mounts
 
-    run_cmd mkfifo /home/omnect/wic.xz || return 1
+    run_cmd mkfifo ${wic_file} || return 1
     run_cmd chown omnect:omnect /home/omnect/wic.xz || return 1
```

Comment:
you introduced ${wic_file}, but there are still usages of `/home/omnect/wic.xz` and `/home/omnect/wic` and `/home/omnect.wic.bmap`.
so why use it in a var only in some locations?

Reply:
wic_file was intended for image file only, so no use for bmap file.
and /home/omnect/wic `/home/omnect/wic` is used together with wic_file during verification, and wic_file adapted accordingly afterward.